### DISCUSSION TITLE
How to! import a local png (or any img file) in the device?

### DISCUSSION THREAD

**Discussion by mrgrlscz** [HASwitchPlate/openHASP#772] [Q&A] [answered]
Hi, i know how to push from HA an image from internet ( or from my HA server) to the plate via https link... but i didn't find the way to load a local image directly in the plate memory...
I saw few example with a `L:/image.png ` source... didn't find the way in the documentation.

**Comment by FreeBear-nc** [comment, marked as answer]
To upload an image or any other file to the plate, use a web browser to connect. Go to File Editor, and click on the Upload button. Follow the prompts to upload a file (it can be any format).
Once stored on the plate, you can display it like this:
```
{"page":1,"id":14,"obj":"img","src":"L:/my_image.png","parentid":2,"auto_size":1,"w":128,"h":128,"x":0,"y":0}
```
Or from HA:
```
    - obj: "p1b14"
      properties:
        "src": "L:/my_image.png"
```

**Reply by mrgrlscz** [reply]
Okay, so.... Very easy.... 
Thank you.
